cofmpy/__init__.py

```diff
@@ -1,36 +1,10 @@
-# -*- coding: utf-8 -*-
```

Comment:
Are you sure it is needed to keep this `cofmpy/__init__.py` file? IMO, it should be present only in the branch `legacy/cofmpy`.